src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/JsonFormat/SearchResultPackagesConverter.cs

```diff
@@ -55,14 +58,20 @@ public override void Write(Utf8JsonWriter writer, IPackageSearchMetadata value,
 
             if (_verbosity == PackageSearchVerbosity.Detailed)
             {
-                writer.WriteString(JsonProperties.Description, value.Description);
+                if (value.Description is not null)
+                {
+                    writer.WriteString(JsonProperties.Description, value.Description);
+                }
 
                 if (value.Vulnerabilities != null && value.Vulnerabilities.Any())
                 {
                     writer.WriteBoolean("vulnerable", true);
                 }
 
-                writer.WriteString(JsonProperties.ProjectUrl, value.ProjectUrl.ToString());
+                if (value.ProjectUrl is not null)
```

Comment:
Can we enable nullable in this class? 

Changes LGTM, just wondering :)

Reply:
I think we can enable null in the class, and I have enabled it now.

Reply:
I know a bunch of the types aren't annotated, but the fact that you enabled it and it didn't raise any issues shows that you did a thorough job with the null checking!